Sandbox/premake5.lua

```diff
@@ -27,6 +27,26 @@ project "Sandbox"
 		"Hazel"
 	}
 
+	filter "system:linux"
+		links
+		{
+			"GLFW",
+			"Glad",
+			"ImGui",
+			"Xrandr",
+			"Xi",
+			"GLU",
+			"GL",
+			"X11",
+			"dl",
+			"pthread",
+			"stdc++fs"
```

Comment:
`stdc++fs` was included in my Hazel Linux branch because `std::filesystem` was used. And the inclusion of this library is only for the transitional C++17 GCC (Clang?) compilers before the filesystem object was moved into the `stdc++` standard library -- which is linked by default.

Reply:
It doesn't matter if stdc++fs is included anyway?

Reply:
I'd say it's fine to leave it since Hazel is a teaching project. `stdc++fs` is a provisional library that is needed for linking on outdated compilers. I have a comment on the Linux branch I've been maintaining to explain that.